templates/helpers/getinventorynumber.js

```diff
@@ -0,0 +1,4 @@
+module.exports = (details) => {
+  var accession = details.find(el => el.key === 'Accession Number');
```

Comment:
is this `.find` being _transpiled_?
(Internet Explorer will choke on it... https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/find#Browser_compatibility )